src/utils/chains.js

```diff
@@ -110,7 +110,7 @@ export const nativeAssetPerChain = {
     name: 'xDAI',
     symbol: XDAI,
     decimals: 18,
-    iconUrl: 'https://tokens.1inch.exchange/0x6b175474e89094c44da98b954eedeac495271d0f.png',
+    iconUrl: 'https://tokens.1inch.exchange/0x6810e776880c02933d47db1b9fc05908e5386b96.png',
```

Comment:
Here's the icon kit for xDai: https://developers.gnosischain.com/media/media-kit#xdai-stable-coin

Reply:
I think this icon is different from Zeplin UI. I set the Gnosis icon as per Zeplin UI.
![ImageMark-LightBG](https://user-images.githubusercontent.com/78720269/186867868-34003ce6-af85-404a-a4a0-ed94efa58751.svg)

![image](https://user-images.githubusercontent.com/78720269/186867992-08b2ce5e-60f2-465c-b4ca-a4c64d3884b1.png)